src/dig/StoreCommands/AddStoreCommand.cs

```diff
@@ -51,14 +54,25 @@ internal sealed class AddStoreCommand()
         var fee = await chiaService.ResolveFee(Fee, Math.Max(serverCoinReserve, mirrorCoinReserve), cts.Token);
         var subscriptions = await dataLayer.Subscriptions(cts.Token);
 
-        var (haveFunds, addedStore) = await storeService.AddStore(Store, subscriptions, mirrorCoinReserve, serverCoinReserve, fee, true, true, url, cts.Token);
+        var (addedStore, addedMirror, addedServerCoin) = await storeService.AddStore(Store, subscriptions, mirrorCoinReserve, serverCoinReserve, fee, true, url, cts.Token);
```

Comment:
is this like the dotNet way to destructure?

Reply:
yup

https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/deconstruct